Library/Formula/awsenv.rb

```diff
@@ -0,0 +1,66 @@
+require "language/go"
+
+class Awsenv < Formula
+  homepage "https://github.com/Luzifer/awsenv"
+  url "https://github.com/Luzifer/awsenv/archive/v0.3.1.tar.gz"
+  head "https://github.com/Luzifer/awsenv.git"
+  sha256 "106ff6acbe72bfc5e8a95a4cfbe49c5c34b4c682775e7072fe740cae8a7d3026"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/Sirupsen/logrus" do
+    url "https://github.com/Sirupsen/logrus",
```

Comment:
Misty wrote the `go_resource` implementation so will defer to @mistydemeo, but I believe `.git` is preferred.